src/index.ts

```diff
@@ -1,4 +1,4 @@
-import type { Format } from 'logform'
+import { Format } from 'logform'
```

Comment:
My IDE was showing an error so I simply removed it. I am guessing it might have been because it was in the middle of creating the declarations.ts. I have added it again.